sdk/keyvault/ci.yml

```diff
@@ -51,3 +51,6 @@ stages:
         - name: azure-keyvault-secrets
           safeName: azurekeyvaultsecrets
           stagingProfileId: 88192f04117501
+        - name: azure-keyvault-certificates
```

Comment:
Any particular reason this is in this PR?

Reply:
Nope, it was accidental, removing it.